src/Symfony/Component/Mailer/EventListener/MessageListener.php

```diff
@@ -119,6 +121,10 @@ private function setHeaders(Message $message): void
     private function renderMessage(Message $message): void
     {
         if (!$this->renderer) {
+            if ($message instanceof TemplatedEmail && ($message->getTextTemplate() || $message->getHtmlTemplate())) {
+                throw new LogicException(sprintf('You must configure a "%s" when a "%s" instance has a text or HTML template set.', BodyRendererInterface::class, get_class($message)));
```

Comment:
```suggestion
                throw new LogicException(sprintf('You must configure a "%s" when a "%s" instance has a text or HTML template set.', BodyRendererInterface::class, get_debug_type($message)));
```

When rendering exception messages, we should prefer `get_debug_type()` over `get_class()`. It plays more nicely with anonymous classes.